src/modules/authentication/api/login.guard.ts

```diff
@@ -62,7 +62,16 @@ export class LoginGuard extends AuthGuard(['jwt', 'oidc']) {
 
             return false;
         }
-
-        return request.isAuthenticated();
+        const isAuthenticated: boolean = request.isAuthenticated();
+        if (isAuthenticated) {
+            this.logger.info(
+                `Benutzer ${request.passportUser?.userinfo.preferred_username} hat sich im Schulportal angemeldet.`,
```

Comment:
Kann man die ID hier auch mit loggen? Benutzer <Benutzername> (<BenutzerID>) hat sich im Schulportal angemeldet.

Reply:
Gibt es an dieser Stelle nicht, ohne größere Anpassungen was out-of-scope wäre. Ich würde sagen, dass das nachgelagert werden kann, wenn man es wirklich braucht. Dann kann man die ID evtl. über Keycloak einsammeln.